### DISCUSSION TITLE
CUTLASS is integrated into TVM

### DISCUSSION THREAD

**Discussion by hwu36** [NVIDIA/cutlass#350] [General]
https://github.com/apache/tvm/commit/541f9f2d8aef9697fd7ccb6a7c0644da273f33b6

In this first commit, Turing and Ampere FP16 tensor core GEMMs are added into TVM.

Great work and thank you very much, TVM community!!!

**Comment by hwu36** [comment]
Now cutlass batched GEMM is integrated, too.

**Comment by masahi** [comment]
Convolution kernels are also integrated. Results and analysis on end to end models are available at https://github.com/apache/tvm/pull/9746.

It looks like TVM + cutlass is faster than TVM + cudnn. I'm not claiming that "cutlass is faster than cudnn on my gpu", it's just that the way we use cudnn is apparently not performing as well as it should be.
